conf/v0.6/general/test_scenario/nccl_test.toml

```diff
@@ -1,174 +1,174 @@
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.10.dependencies]
-    start_post_comp = { name = "Tests.9", time = 0 }
-
-[Tests.11]
-  name = "nccl_test_all_reduce"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.11.dependencies]
-    start_post_comp = { name = "Tests.10", time = 0 }
-
-[Tests.12]
-  name = "nccl_test_all_gather"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.12.dependencies]
-    start_post_comp = { name = "Tests.11", time = 0 }
-
-[Tests.13]
-  name = "nccl_test_reduce_scatter"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.13.dependencies]
-    start_post_comp = { name = "Tests.12", time = 0 }
-
-[Tests.14]
-  name = "nccl_test_alltoall"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.14.dependencies]
-    start_post_comp = { name = "Tests.13", time = 0 }
-
-[Tests.15]
-  name = "nccl_test_bisection"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.15.dependencies]
-    start_post_comp = { name = "Tests.14", time = 0 }
-
-[Tests.16]
-  name = "nccl_test_all_reduce"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.16.dependencies]
-    start_post_comp = { name = "Tests.15", time = 0 }
-
-[Tests.17]
-  name = "nccl_test_all_gather"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.17.dependencies]
-    start_post_comp = { name = "Tests.16", time = 0 }
-
-[Tests.18]
-  name = "nccl_test_reduce_scatter"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.18.dependencies]
-    start_post_comp = { name = "Tests.17", time = 0 }
-
-[Tests.19]
-  name = "nccl_test_alltoall"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.19.dependencies]
-    start_post_comp = { name = "Tests.18", time = 0 }
-
-[Tests.20]
-  name = "nccl_test_bisection"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.20.dependencies]
-    start_post_comp = { name = "Tests.19", time = 0 }
-
-[Tests.21]
-  name = "nccl_test_all_reduce"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.21.dependencies]
-    start_post_comp = { name = "Tests.20", time = 0 }
-
-[Tests.22]
-  name = "nccl_test_all_gather"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.22.dependencies]
-    start_post_comp = { name = "Tests.21", time = 0 }
-
-[Tests.23]
-  name = "nccl_test_reduce_scatter"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.23.dependencies]
-    start_post_comp = { name = "Tests.22", time = 0 }
-
-[Tests.24]
-  name = "nccl_test_alltoall"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.24.dependencies]
-    start_post_comp = { name = "Tests.23", time = 0 }
-
-[Tests.25]
-  name = "nccl_test_bisection"
-  num_nodes = "2"
-  time_limit = "00:20:00"
-  [Tests.25.dependencies]
-    start_post_comp = { name = "Tests.24", time = 0 }
+[[Tests]]
+name = "nccl_test_all_reduce"
+num_nodes = 2
+time_limit = "00:20:00"
+
+[[Tests]]
+name = "nccl_test_all_gather"
+num_nodes = 2
+time_limit = "00:20:00"
+[Tests.dependencies]
+start_post_comp = { name = "Tests.1", time = 0 }
```

Comment:
There is no name called "Tests.1". I believe this is not a valid dependency.

Reply:
Thanks! Reworked it so it works again :)